artman/cli/converter.py

```diff
@@ -0,0 +1,306 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Artman config converter.
+
+It converts the legacy artman config file into the new format. This CLI can be
+removed after all current artman users migrate to the new artman config format.
+"""
+
+from __future__ import absolute_import
+import argparse
```

Comment:
Also:

  * Empty line after `from __future__ import absolute import`.
  * Empty line before `import yaml`, which is not in the standard library.